src/ascii.rs

```diff
@@ -60,6 +61,50 @@ where
         Ok(buffer)
     }
 
+    /// Returns values for multiple keys in a single call as a `HashMap` from keys to found values.
+    /// If a key is not present in memcached it will be absent from returned map.
+    pub async fn get_multi<'a, K: Display>(&'a mut self, keys: Vec<&'a K>) -> Result<HashMap<String, Vec<u8>>, Error> {
+        // Send command
+        self.io.write_all("GET".as_bytes()).await?;
+        for k in keys {
+            self.io.write(&[32; 1]).await?;
+            self.io.write_all(k.to_string().as_bytes()).await?;
```

Comment:
`k.into_bytes()` but WriteAll shouldn't require owned Vec, so `k.as_bytes()` should work without extra copy

Reply:
The problem is k is Display which requires conversion to string first.